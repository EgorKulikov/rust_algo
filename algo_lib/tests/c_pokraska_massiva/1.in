5
5
1 2 3 4 5
3
10 5 15
3
100 10 200
10
9 8 2 6 6 2 8 6 5 4
2
1 3
