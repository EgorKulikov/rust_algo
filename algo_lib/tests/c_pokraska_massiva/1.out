2
0
100
0
3
